components/infieldbutton/stories/infieldbutton.stories.js

```diff
@@ -1,80 +1,86 @@
 import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
+import { uniqueUiIconBaseNames } from "@spectrum-css/icon/stories/utilities.js";
+import { Sizes, withDownStateDimensionCapture } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isActive, isDisabled, isFocused, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
+import { isActive, isDisabled, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
 import metadata from "../dist/metadata.json";
 import packageJson from "../package.json";
 import { InfieldButtonGroup } from "./infieldbutton.test.js";
-import { Template } from "./template.js";
+import { InfieldButtonGroupVariant, InfieldButtonIcons, Template } from "./template.js";
 
 /**
- * The in-field button component is a button used inside a text field.
+ * In-field buttons are used to represent actions within input fields. They’re currently used inside the [combo box,](/docs/components-combobox--docs) [number field,](/docs/components-stepper--docs) and [search field.](/docs/components-search--docs)
```

Comment:
Similar to the changeset, we'll need to update this description since only number field actually uses the infield buttons (combobox uses picker button and search field is a clear button, both different components currently).